src/c/inteldeflater/IntelDeflater.c

```diff
@@ -1,247 +0,0 @@
-/*
```

Comment:
Why did this get removed?  The IntelDeflator speeds up deflation and is really quite useful.  Do not remove.